std_msgs/Float64 distance
---
std_msgs/Int8 status
